tools/autograd/derivatives.yaml

```diff
@@ -8,6 +8,12 @@
 #     Note that a single gradient entry can specify the gradient
 #     formula for multiple input names, by specifying a key
 #     "input1, input2" (see atan2 for an example).
+#   - Optional entry with key 'output_differentiability' and value a list of the
```

Comment:
So... this means that we should forbid people from declaring native functions whose has an argument name `output_differentiability`, right? :)

Reply:
Technically no, because each entry in this yaml has `name: native_fn_declaration`. The name of the function isn't a key. But I can't think of a scenario where such a function name would make sense. :D